extensions/wikia/Recirculation/styles/mixed-content-footer.scss

```diff
@@ -263,3 +263,109 @@
 		margin-left: 12px;
 	}
 }
+
+.mcf-card-discussions {
+	border: 1px solid $wds-color-faint-gray;
+	background-color: $wds-color-white;
+	max-width: 334px;
+	height: 608px;
+
+	&__header {
+		@include flexbox();
+		@include justify-content(space-between);
+		color: $wds-color-black;
+		border-bottom: 1px solid $wds-color-faint-gray;
+		font-size: 18px;
+		font-weight: 600;
+		padding: 16px;
+		position: relative;
+	}
+
+	&__link {
+		@include align-items(center);
+		@include flexbox();
+		font-size: 12px;
+		font-weight: bold;
+		text-align: right;
+		color: $wds-color-light-blue;
+		text-decoration: none;
+	}
+
+	&__link-icon {
+		margin-left: 6px;
+		transform: rotate(180deg);
+	}
+
+	ul {
+		@include scrollable-list(555px, $wds-color-white);
+	}
+
+	&__item {
+		@include align-items(flex-start);
+		@include flexbox();
+		@include flex-direction(column);
+		border-bottom: 1px solid $wds-color-faint-gray;
+		margin: 0 16px;
+		min-height: 70px;
+		min-width: 302px;
+
+		a {
+			text-decoration: none;
+			width: 100%;
+		}
+	}
+
+	&__avatar {
+		border: 2px solid $wds-color-faint-gray;
```

Comment:
maybe we can use `.wds-avatar`?
Look here: https://github.com/Wikia/app/blob/4da98907e741332bcfb42ed639262773e56b2e1a/extensions/wikia/Forum/templates/Forum_forumRelatedThreads.php#L12-L12 how we did it for forum module